static/src/js/components/Errors/NotFound.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import uuidv4 from 'uuid/v4'
+
+import LoggerRequest from '../../util/request/loggerRequest'
+
+const NotFound = ({ location }) => {
+  // eslint-disable-next-line global-require
+  require('./ErrorBoundary.scss')
+
+  const guid = uuidv4()
+
+  const requestObject = new LoggerRequest()
+  requestObject.log({
+    error: {
+      guid,
+      message: '404 error',
```

Comment:
Do we know what wasn't found? API gateway returns an appropriate message that we should display if possible. Also, I prefer `404 Not Found` 🤷‍♂